puffin/src/lib.rs

```diff
@@ -405,8 +339,9 @@ impl ThreadProfiler {
         new_id
     }
 
+    /// Register a named scope.
     #[must_use]
-    pub fn register_scope(
+    pub fn register_named_scope(
```

Comment:
This is a breaking change that slipped in – was that intentional?

Reply:
this function should be hidden from API as it should not be called by users. I will revert it in this PR but follow up with general encapsulation PR